### PR TITLE
feat: support emojis

### PR DESCRIPTION
**Pull request by moul**
Before:
Screenshot - Manfred 2020-11-09 at 15 10 05

After:
Screenshot - Manfred 2020-11-09 at 15 09 58

h/t @zxxma

### CODE REVIEW

**Comment by zxxma** [comment]
dope! 👍 SGTM&LGTM! 😄

File: static/_assets/js/script.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-if (true || !window.sgtm.dev_mode) {
+OBif (true || !window.sgtm.dev_mode) {
```
Comment (n0izn0iz) [inline static/_assets/js/script.js]: typo ?

Comment (moul) [reply static/_assets/js/script.js]: completely 👍

**Review by n0izn0iz — APPROVED** [review]
lgtm

**Pull request merged**